app/assets/javascripts/mumuki_laboratory/application/kids.js

```diff
@@ -173,13 +173,17 @@ mumuki.load(function () {
     },
 
     _showMessageOnCharacterBubble: function (data) {
-      var $bubble = mumuki.kids._getCharacterBubble();
+      let $bubble = mumuki.kids._getCharacterBubble();
```

Comment:
Prefer const over let if we aren't mutating them afterwards

Reply:
I will fix that in #1400 , since it involves some big refactors